### PR TITLE
fix: white box in quick input page

### PR DESCRIPTION
**Pull request by iamkryptonite**
Signed-off by: @iamkryptonite thesatyakidas.00220033@gmail.com
Fixes : #393 

Bench_routes
Bench_routes

### CODE REVIEW

**Comment by iamkryptonite** [comment]
@muskankhedia this PR is ready for review.

**Review by muskankhedia — CHANGES_REQUESTED** [review]
@iamkryptonite This looks good. Can you make the dark-mode changes for the label field as well?

We can change the text colour to white in the dark mode that will help to view the contents in an effective way.
Screenshot 2021-03-20 at 6 36 25 PM

Also. can you remove the white space area below it? The white space is actually for the scroll bar but the labels can be displayed in the next row.
Screenshot 2021-03-20 at 6 36 36 PM

**Comment by iamkryptonite** [comment]
Okay sure.

**Comment by iamkryptonite** [comment]
@muskankhedia I didn't quite understand your requested changes. Do you want me to completely remove the scroll ?

**Comment by muskankhedia** [comment]
> @muskankhedia I didn't quite understand your requested changes. Do you want me to completely remove the scroll ?

Yes, and as scroll bar will be disabled the labels will appear on a single line which won't look good. So you can design it as a `container` or `div` element containing all the labels.

**Comment by iamkryptonite** [comment]
@muskankhedia 
Bench_routes2
Bench_routes1
Bench_routes

Also I think that if we make 'Add labels' into a button or add a '+' sign it will be easier to understand that it is a button for adding labels.

**Comment by muskankhedia** [comment]
> that if we make 'Add label

Sure, that will look good. Will it be possible for you to make the changes?

**Comment by iamkryptonite** [comment]
> > that if we make 'Add label
> 
> Sure, that will look good. Will it be possible for you to make the changes?

@muskankhedia  does this look okay?
Bench_routes3

**Comment by sachdeva-shrey** [comment]
Hey @iamkryptonite @muskankhedia, this looks like a duplicate of https://github.com/bench-routes/bench-routes/pull/359.

**Comment by iamkryptonite** [comment]
> Hey @iamkryptonite @muskankhedia, this looks like a duplicate of #359.

Correct. Maybe I should leave the "Add labels" button as it is and later on when your PR is merged the changes will be reflected accordingly.

**Comment by muskankhedia** [comment]
> Hey @iamkryptonite @muskankhedia, this looks like a duplicate of #359.

Hello Shrey, I am sorry, I think I missed that PR and didn't know the design refactoring was already done. Thank you for pointing it out.

File: dashboard/v1.1/src/pages/Input/components/MaterialLabelSelector.tsx [hunk 1]
```diff
@@ -157,8 +157,7 @@ export default function MaterialLabelSelector(props: LabelSelectorType) {
               renderTags={() => null}
               noOptionsText={
                 <div className={classes.noOption} onMouseDown={handleAddLabel}>
-                  Add new label
-                  <div className={classes.tag}> {truncate(labelValue, 10)}</div>
```
Comment (muskankhedia) [inline dashboard/v1.1/src/pages/Input/components/MaterialLabelSelector.tsx:161]: @iamkryptonite Can you please explain why have you removed this?

**Review by muskankhedia — CHANGES_REQUESTED** [review]

File: dashboard/v1.1/src/pages/Input/components/MaterialLabelSelector.tsx (hunk 1 above)
Comment (iamkryptonite) [reply dashboard/v1.1/src/pages/Input/components/MaterialLabelSelector.tsx:161]: the tag class was for the labels that were selected and they had padding and margin assigned to them,which made the new label name shift downward. 

Before:

After:

Comment (muskankhedia) [reply dashboard/v1.1/src/pages/Input/components/MaterialLabelSelector.tsx:161]: Okay

**Review by muskankhedia — APPROVED** [review]
Thank you for the PR!

**Comment by muskankhedia** [comment]
@iamkryptonite Can you please rebase it with the master branch and fix the merge conflicts?

**Pull request merged**